continuous_integration/install.sh

```diff
@@ -76,3 +76,4 @@ if [ -z "$NUMPY_VERSION" ]; then
 fi
 
 python setup.py install
+pip install nose-exclude
```

Comment:
It's to avoid dill's own tests which fail for some reason, might be related to converting to relative imports but I don't think it matters.